_includes/inline-style.css

```diff
@@ -76,8 +76,9 @@ main *, .hero * {
 
 body {
   background-color: var(--c-bg);
-  min-height: 100vh;
+  height: 100vh;
```

Comment:
feels like this will have the effect that the footer will always be on screen? instead of being off screen on long pages?